jsm.go

```diff
@@ -727,6 +735,78 @@ func (js *js) StreamInfo(stream string, opts ...JSOpt) (*StreamInfo, error) {
 	return resp.StreamInfo, nil
 }
 
+func (js *js) StreamContainedSubjects(stream string, filter ...string) (map[string]uint64, error) {
+	if err := checkStreamName(stream); err != nil {
```

Comment:
You need to pass JSOpt and do like StreamInfo in that use will be able to override the ttl for the request (especially an API like this that can take time to collect all results). The filter could then be made an option.